packages/office-ui-fabric-react/src/components/SearchBox/docs/new/SearchBoxBestPractices.md

```diff
@@ -0,0 +1,24 @@
+### Layout
+
+- Use a visual separator to group similar or related autocomplete suggestions.
+- If possible, provide a preview for autocomplete suggestions to help people quickly determine if the suggested result is helpful. For example, an image preview or file title.
+- Don't build a custom search control based on the default text box or any other control.
+
+[Sketch toolkit]()
+
+[Figma toolkit]()
```

Comment:
These are new doc pages in progress (not getting published yet), and I assume the links will be filled in before publishing. I'm guessing it would be helpful for designers to have toolkit links inline instead of having to dig it up from the Resources page.